apps/web/src/pages/workflow/views/editor/components/config-panel/components/service-entity-call/index.tsx

```diff
@@ -0,0 +1,131 @@
+            const entityItem = entityFilterList[0];
+            const { error, res } = await getEntityChildren({
+                id: entityItem.entity_id,
+            });
+            if (error) {
+                resetList([]);
+            } else {
+                resetList(
+                    res.map((item: EntityItem) => {
+                        return {
+                            key: item.entity_key,
+                            name: item.entity_name,
+                            type: item.entity_value_type,
+                        };
+                    }),
+                );
+            }
+        } else {
+            resetList([]);
+        }
+        setInnerValue(pre => ({ ...pre, serviceValue: value }));
+    };
+    const renderInputParam = useMemo(() => {
+        if (list.length) {
+            return (
+                <div className="ms-service-entity-call-input">
+                    <Divider className="ms-service-entity-call-input-divider" />
+                    <span className="ms-service-entity-call-title">
+                        {getIntlText('workflow.node.service_call_Input_title')}
+                    </span>
+                    {list.map((item, index) => {
+                        return (
+                            <div key={item.key} className="param-item">
+                                <div className="param-item-title">
+                                    <span className="param-item-name">{item.name}</span>
+                                    <span className="param-item-type">{item.type}</span>
+                                </div>
+                                <ParamInputSelect
+                                    required={required}
+                                    value={item?.value}
+                                    onChange={data => {
+                                        replace(index, { ...item, value: data });
+                                    }}
+                                />
+                            </div>
+                        );
+                    })}
+                </div>
+            );
+        }
+        return null;
+    }, [list]);
+    useLayoutEffect(() => {
+        setInnerValue(pre => ({ ...pre, paramList: list }));
+    }, list);
+    return (
+        <div className="ms-service-entity-call">
+            <span className="ms-service-entity-call-title">
+                {getIntlText('workflow.node.service_call_service_title')}
+            </span>
+            <EntitySelect
+                label={getIntlText('common.label.service')}
+                required={required}
+                disabled={disabled}
+                value={innerValue?.serviceValue}
+                filterModel={filterModel}
+                onChange={data => {
+                    handlerChange(data);
+                }}
+            />
+            {renderInputParam}
+        </div>
+    );
+};
+export default ServiceEntityCall;
```

Comment:
According to the back-end design, this resource should be split into two components, one is `EntitySelect`, and the other may be named `ServiceParamAssignInput`.